adoc/Release-Notes.adoc

```diff
@@ -33,6 +33,75 @@ endif::[]
 [id='sec.major-change']
 == Major Changes
 
+[id='sec.1_4_1']
+=== Release 1.4.1, July 2019
+
+[id='sec.1_4_1.new']
+==== What Is New?
+
+* {scf} has been updated to version 2.17.1.
+* Added option to deploy UAA from the SCF Helm chart. In new CAP deployments that do not require a separate UAA deployment, setting `enable.uaa=true` in the Helm values automatically creates a UAA StatefulSet within the SCF deployment. This removes the manual step of copying the UAA_CA_CERT during setup. In this configuration UAA uses the same database as SCF, reducing the memory footprint of the cluster. This option should be left as `false` (default) for upgrades of existing CAP clusters.
+
+[id='sec.1_4_1.feature']
+==== Features and Fixes
+
+* Set the default value of `AZ_LABEL_NAME` to `failure-domain.beta.kubernetes.io/zone`
+* Simplified service accounts and pod security policies
+* Switched to log-cache for container metrics
+* Implemented a patch to squash Cloud Controller database migrations
+* Fixed version and SHA1 of `cf-mysql-release` tied to version 36.15.0
+* Fixed TLS issues in `log-cache`
+
+* Includes these {cf} component versions:
+
+** app-autoscaler: 1.2.1
+** bits-service: 2.26.0
+** bpm: 1.0.0
+** capi: 1.79.0
+** cats: 7.11
+** cf-deployment: 7.11
+** cf-mysql: 36.15.0
+** cf-routing: 0.187.0
+** cf-sle12: 1.75.11
+** cf-smoke-tests: 40.0.51
+** cf-syslog-drain: 10.0
+** cf-usb: 1.0.1
+** cflinuxfs2: 1.281.0
+** cflinuxfs3: 0.108.0
+** credhub: 2.1.2
+** diego: 2.30.0
+** eirini: 0.0.4
+** garden-runc: 1.19.1
+** groot-btrfs: 1.0.4
+** kubectl: 1.9.6
+** loggregator: 105.2
+** loggregator-agent: 3.9
+** nats: 26
+** nfs-volume: 1.5.2
+** postgres-release: 26
+** scf-helper: 1.0.2
+** statsd-injector: 1.9.0
+** uaa: 68.0
+* Buildpacks:
+** binary-buildpack: 1.0.32
+** dotnet-core-buildpack: 2.2.12
+** go-buildpack: 1.8.41
+** java-buildpack: 4.19.1 
+** nginx-buildpack: 1.0.14
+** nodejs-buildpack: 1.6.51
+** php-buildpack: 4.3.77
+** python-buildpack: 1.6.34
+** ruby-buildpack: 1.7.40
+** staticfile-buildpack: 1.4.43
+
+[id='sec.1_4_1.issue']
+==== Known Issues
+
+* `cf-deployment` 7.11 is the final Cloud Foundry version that supports the `cflinuxfs2` stack. The `cflinuxfs2` and `sle12` stacks are deprecated in favor of `cflinuxfs3` and `sle15` respectively. Start planning to migrate applications to those stacks for futureproofing, as these stacks will be removed in a future release. The Stack Auditor plugin for `cf` can help with this migration (see https://docs.cloudfoundry.org/adminguide/stack-auditor.html). 
```

Comment:
* I'd suggest `final` -> `last` because that sounds clearer to me. (But that is just gut feeling from a non-native speaker.)

Reply:
No wording guidance from https://github.com/cloudfoundry/cf-deployment/releases/tag/v7.11.0 but I'm OK with `last`.